config/main.py

```diff
@@ -4413,6 +4549,107 @@ def delete(ctx):
     sflow_tbl['global'].pop('agent_id')
     config_db.set_entry('SFLOW', 'global', sflow_tbl['global'])
 
+#
+# set ipv6 link local mode on a given interface
+#
+def set_ipv6_link_local_only_on_interface(config_db, interface_dict, interface_type, interface_name, mode):
+
+    curr_mode = config_db.get_entry(interface_type, interface_name).get('ipv6_use_link_local_only')
+    if curr_mode is not None:
+        if curr_mode == mode:
+            return
+    else:
+        if mode == "disable":
+            return
+
+    if mode == "enable":
+        config_db.mod_entry(interface_type, interface_name, {"ipv6_use_link_local_only": mode})
```

Comment:
will this create a new entry if not already present?

Reply:
Yes, it is.